example-config

```diff
@@ -5,6 +5,10 @@
 # Move 'config' into a task folder 
 
 
+# Values begining with ! or @ are reserved and have special meaning:
+# - Only ! value is !unset, which is always invalid
```

Comment:
What is it for?

Reply:
This is for #135. It does not make sense yet, but you should be able to force task author to set a key.

Reply:
As currently implemented, using !unset has the same effect on value resolution as simply not specifing the key, except that is skips checking other candidates. It will still search lower priority configs.

Reply:
I need to break twice, sorry..